python-sdk/pachyderm_sdk/config.py

```diff
@@ -6,9 +6,10 @@
 from typing import Dict, Optional, Union
 
 from .errors import ConfigError
+from .constants import CONFIG_PATH_LOCAL
 
 class ConfigFile:
-    def __init__(self, config_file: Union[Path, str]):
+    def __init__(self, config_file: Union[Path, str]=CONFIG_PATH_LOCAL):
```

Comment:
I wouldn't add a default here. In practice I've found when args are passed through a chain of functions, only have a default arg when they first appear results in the least bugs. In this case I would default it at the Client and not in Config